docs/cross-compilation.md

```diff
@@ -113,43 +131,49 @@ scala_library(
 See complete documentation in the [scala_cross_version_select.bzl](/scala/scala_cross_version_select.bzl) file
 
 #### Manually
+
 An example usage of `select()` to provide custom dependency for specific Scala version:
-```starlark
+
+```py
 deps = select({
     "@io_bazel_rules_scala_config//:scala_version_3_3_1": [...],
     ...
 })
 ```
 
 For more complex logic, you can extract it to a `.bzl` file:
-```starlark
+
+```py
 def srcs(scala_version):
     if scala_version.startswith("2"):
         ...
     ...
 ```
+
 and then in the `BUILD` file:
-```starlark
-load("@io_bazel_rules_scala//:scala_cross_version.bzl", "version_suffix")
-load("@io_bazel_rules_scala_config//:config.bzl", "SCALA_VERSIONS")
+
+```py
 load("....bzl", "srcs")
+load("@io_bazel_rules_scala_config//:config.bzl", "SCALA_VERSIONS")
+load("@rules_scala//:scala_cross_version.bzl", "version_suffix")
 
 scala_library(
     ...
     srcs = select({
         "@io_bazel_rules_scala_config//:scala_version" + version_suffix(v): srcs(v)
```

Comment:
I think there are still a lot of "io_bazel_rules_scala" instances laying around in the docs.

Reply:
Not really, though I did double check and found two more that I've since updated in https://github.com/bazelbuild/rules_scala/pull/1703/commits/5bb45294a140ae45c372d4ea620692b7f8a0225b (one in `README.md`, another in `.github/workflows/workspace_snippet.sh`).

As mentioned in that commit message, the remaining references are actually prefixed `io_bazel_rules_scala_` (notice the trailing `_`). Most of them are private, with the exception of `@io_bazel_rules_scala_config`, which is a documented public interface.

I also mention that we _could_ change this to `@rules_scala_config` as part of the breaking changes for v7.0.0. If @simuons and/or @liucijus say the word, I'm happy to do that in a separate pull request.

Reply:
Thanks, makes sense.

Reply:
@mbland I'm ok with renaming `@io_bazel_rules_scala_config` since there will be breaking changes as you mentioned and there is no reason for keeping label misaligned.

Reply:
@simuons Thanks, I'll take care of that in a follow-up PR. That'll be another sweeping, albeit straightforward change similar to #1696, and this current PR has already gotten bit enough.

Reply:
Preview: [rules-scala-config-repo-name in mbland/rules_scala](https://github.com/mbland/rules_scala/tree/rules-scala-config-repo-name) (specifically https://github.com/mbland/rules_scala/commit/d95c0e7ba6d1500df406e688e62d06c3229e85c0).

It wasn't _as_ big a diff as #1696, but still probably worth its own PR.